### PR TITLE
[#72] Remove unused code from catcher models

### PR DESCRIPTION
**Pull request by Eclipse-Dominator**
### Summary:
Part of PRs for #72 
Depends on #145 (Review this only after this PR is merged)

### Changes Made:
- Removes unused models
- Removes unused functions from Issues
- Removes unused package imports

### Commit Message:

```
As WATcher is ported from CATcher, there are a lot of models and methods
that are not used and are unrelated to WATcher.

Let's remove most of the unused code in core/models
```

### CODE REVIEW

**Review by gycgabriel — APPROVED** [review]
LGTM

**Comment by gycgabriel** [comment]
I noticed some of the changes are formatting, do we have husky pretty-quick working on WATcher?

File: src/app/issues-viewer/issues-viewer.component.html [hunk 1]
```diff
@@ -1,11 +1,6 @@
 <div>
-  <div class="loading-spinner" *ngIf="this.phaseService.isChangingRepo | async; else elseBlock;">
-    <mat-progress-spinner 
-      color="primary"
-      mode="indeterminate"
-      diameter="50"
-      strokeWidth="5">
-    </mat-progress-spinner>
+  <div class="loading-spinner" *ngIf="this.phaseService.isChangingRepo | async; else elseBlock">
+    <mat-progress-spinner color="primary" mode="indeterminate" diameter="50" strokeWidth="5"> </mat-progress-spinner>
```
Comment (gycgabriel) [inline src/app/issues-viewer/issues-viewer.component.html:3]: And this

File: src/app/issues-viewer/card-view/card-view.component.css [hunk 2]
```diff
@@ -52,11 +52,9 @@ div.column-header .mat-card-header {
...
 
   /* Ref: https://css-scroll-shadows.vercel.app */
-  background: 
-    linear-gradient(white 33%, transparent),
-    linear-gradient(transparent, white 66%) 0 100%,
-    radial-gradient(farthest-side at 50% 0, rgba(0,0,0, 0.5), transparent),
-    radial-gradient(farthest-side at 50% 100%, rgba(0,0,0, 0.5), transparent) 0 100%;
+  background: linear-gradient(white 33%, transparent), linear-gradient(transparent, white 66%) 0 100%,
+    radial-gradient(farthest-side at 50% 0, rgba(0, 0, 0, 0.5), transparent),
+    radial-gradient(farthest-side at 50% 100%, rgba(0, 0, 0, 0.5), transparent) 0 100%;
```
Comment (gycgabriel) [inline src/app/issues-viewer/card-view/card-view.component.css:57]: E.g. this

**Comment by Eclipse-Dominator** [comment]
> I noticed some of the changes are formatting, do we have husky pretty-quick working on WATcher?

We do have it working, it seems pretty-quick is reformatting those lines into those mentioned

**Comment by gycgabriel** [comment]
Hmm, but is it doing that with each commit-push automatically? Odd that the dev that did the card-view-component didn't have the pretty quick done already

**Review by gycgabriel — APPROVED** [review]
LGTM

**Pull request merged**